react/components/Promotion/EligibilitySection/Coupons/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react'
+import { Button, ToastConsumer } from 'vtex.styleguide'
+import { FormattedMessage, useIntl } from 'react-intl'
+import { useQuery, useMutation } from 'react-apollo'
+
+import createCoupon from '../../../../graphql/createCoupon.graphql'
+import couponsFromUTMs from '../../../../graphql/couponsFromUTMs.graphql'
+import NewCouponModal from './NewCouponModal'
+import CouponsList from './CouponsList'
+
+export type CouponInput = {
+  couponCode: string
+  utmSource: string
+  utmCampaign: string
+  maxItemsPerClient: number
+  daysUntilRenewal: number
```

Comment:
```suggestion
  maxItemsPerClient?: number
  daysUntilRenewal?: number
```

Aren't those values optional?